validation/jinja2/validation/advanced_search.html

```diff
@@ -0,0 +1,56 @@
+{#
+    # Displays search form to perform an advanced search
+#}
+   
+{% extends 'validation/_base.html' %}
+{% import 'validation/_macros.html' as macros %}
+
+{% block content %}
+<script>
+    function getSelectedSpeakers() {
```

Comment:
I have SEVERAL coding suggestions for this method, but I think it may not even be needed at all! Having a bunch of `<input type="checkbox">` with the same name will submit something like this to Django:

```
?transcription=&translation=&status=all&speaker-options=all&speaker-options=MAR&speaker-options=JER&speaker-options=ROS2
```

Which then can be used in accessed in Django as:

```python
>>> request.GET.getlist("speaker-options")
["all", "MAR", "JER", "ROS2"]
```

No JavaScript needed! And as well all know, the best JavaScript is no JavaScript at all! 😇

Reply:
Oh this is beautiful! I was not happy with that function either, but I thought it was a necessary evil or sorts. Consider it gone!